auto-gen.sh

```diff
@@ -99,6 +113,8 @@ set -e
 
 if [ "$2" = "generated/mocks" ]; then
     mocks_clear "*_mock.go"
```

Comment:
Should this be `remove_matching_files ` now?

Reply:
Yeah good catch ta, missed this during merge conflicts.